presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -186,17 +191,42 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
     {
         RaptorTableHandle handle = checkType(tableHandle, RaptorTableHandle.class, "tableHandle");
         SchemaTableName tableName = new SchemaTableName(handle.getSchemaName(), handle.getTableName());
-        List<ColumnMetadata> columns = dao.listTableColumns(handle.getTableId()).stream()
-                .map(TableColumn::toColumnMetadata)
-                .filter(isSampleWeightColumn().negate())
-                .collect(toCollection(ArrayList::new));
+        List<TableColumn> columns = dao.listTableColumns(handle.getTableId());
```

Comment:
Name this `tableColumns` and keep the name `columns` for the `List<ColumnMetadata>` below. Then the `columns.add()` lines won't change.